packages/dds/tree/src/test/feature-libraries/defaultChangeFamily.spec.ts

```diff
@@ -45,16 +59,45 @@ function assertDeltasEqual(actual: Delta.Root[], expected: Delta.Root[]): void {
     }
 }
 
+function setup(data?: JsonableTree) {
```

Comment:
I think this could use a return type annotation, and possibly a slightly more descriptive name.

Reply:
I tried to improve things a bit but this function really should not exist (it will go away at part of my PR that makes it easier to write tests). Happy to make more adjustements if you have more thoughts.

Reply:
I tried to improve things a bit but this function really should not exist (it will go away at part of my PR that makes it easier to write tests). Happy to make more adjustements if you have more thoughts.